src/main/scala/carldata/series/TimeSeries.scala

```diff
@@ -199,6 +199,45 @@ case class TimeSeries[V](idx: Vector[LocalDateTime], ds: Vector[V]) {
     new TimeSeries(index, vs)
   }
 
+  /** Merges series A into B */
+  def merge(ts: TimeSeries[V]): TimeSeries[V] = {
+    val res: mutable.ListBuffer[(LocalDateTime, V)] = ListBuffer()
+
+    @tailrec
+    def insert(xs: TimeSeries[V], ys: TimeSeries[V]): TimeSeries[V] = {
```

Comment:
I think it would be better if this function will have the following declaration:
```scala
def insert(xs: Vector[(LocalDateTime, V)], ys: Vector[(LocalDateTime, V)]): Vector[(LocalDateTime, V)]
```

Look at function "step, resample, addMissing" etc. They all work this way

This way we won't have to create TimeSeries objects in each iteration